src/core/utils/default_nav_groups.ts

```diff
@@ -64,16 +64,16 @@ const defaultNavGroups = {
     }),
     order: 5000,
   },
-  analytics: {
+  essentials: {
     id: 'analytics',
```

Comment:
Nit: Shall we add some comments here to show why we still using analytics instead of essentials?